README.md

```diff
@@ -29,7 +29,7 @@ $ curl 0.0.0.0:8000/versions/default/foo-controller
 {"container":"clux/controller","name":"foo-controller","version":"latest"}
 ```
 
-and its metrics (currently disabled due to actix upgrade issues):
+and its metrics (previously disabled due to `actix-web` upgrade issues, now omitted due to missing Axum/`prometheus` integration):
```

Comment:
can just drop this line entirely, remove the metrics instructions further down in the README then.
i can open an issue on getting something equivalent to actix-web-prom back

Reply:
c563a4a1d54b8bc224c383644679ee20d7ceb63a